src/UnderlineNav/UnderlineNavLink.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import Box from '../Box'
+import {merge} from '../sx'
+import {get} from '../constants'
+
+export const UnderlineNavLink = ({
+  sx: sxProp = {},
+  href = '#',
+  children,
+  selected = false,
+  leadingIcon: LeadingIcon,
+  ...props
+}) => {
+  const linkStyles = {
+    display: 'inline-flex',
+    color: 'fg.default',
+    textAlign: 'center',
+    borderBottom: '2px solid transparent',
+    borderColor: selected ? 'primer.border.active' : 'transparent',
+    textDecoration: 'none',
+    paddingX: get('space.2'),
+    paddingY: get('space.3'),
+    marginRight: get('space.3'),
+    fontSize: get('fontSizes.1'),
```

Comment:
`get()` shouldn't be necessary since this object is passed to the sx prop

Reply:
I'm not sure. whats the alternative? It doesn't seem to work without the get.